webapp/app/templates/components.html

```diff
@@ -35,52 +35,64 @@
     {% endif %}
 {%- endmacro %}
 
-{% macro checkbox(field, classes="") %}
+{% macro checkbox(field, classes="", position_details_after=False) %}
     <div class="form-row checkbox">
         {{ _field(field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
-        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
+        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0", position_details_after=position_details_after) }}
     </div>
 {% endmacro %}
 
-{% macro consent_box(field, classes="") %}
+{% macro consent_box(field, classes="", position_details_after=False) %}
     <div class="form-row checkbox consent-box col-lg-10">
         {{ _field(field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
-        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
+        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0", position_details_after=position_details_after) }}
     </div>
 {% endmacro %}
 
-{% macro radio_field(field) -%}
+{% macro radio_field(field, position_details_after=False) -%}
     <fieldset>
-        {{ field_label(field) }}
+        {{ field_label(field, position_details_after=position_details_after) }}
         {% for subfield in field %}
             <div>
                 {{ subfield(class="") }}
-                {{ field_label(subfield, class="col-sm-10 col-form-label") }}
+                {{ field_label(subfield, class="col-sm-10 col-form-label", position_details_after=position_details_after) }}
             </div>
         {% endfor %}
+        {%- if position_details_after %}
+            {%- set detail = field.render_kw['detail'] -%}
+            {%- if detail %}
+                {{ details(detail.title, [detail.text], details_id=field.id) }}
+            {% endif %}
+        {% endif %}
         {{ field_errors(field) }}
```

Comment:
Should we maybe make this a separate macro? like _detail() or something? then we'd only need to change one place.

Reply:
And is it a decision to add it do the separated field but not to consent or checkbox fields?

Reply:
> Should we maybe make this a separate macro? like _detail() or something? then we'd only need to change one place.

But how would that work? If we do not set it at this point we do not set it?

They do not have a details component and therefore I did not add one.

Reply:
Okay, I've added a macro for that